shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/sql/statement/dal/SetStatement.java

```diff
@@ -29,4 +31,6 @@
 public final class SetStatement extends DALStatement {
     
     private VariableSegment variable;
```

Comment:
Can you consider about rename VariableSegment? We can not understand the relationship with  VariableSegment and VariableValueSegment

Reply:
re-code  re-design